config.json

```diff
@@ -34,7 +34,7 @@
       "description": "Specifies the distro rid for Unix OS.",
       "valueType": "property",
       "values": [],
-      "defaultValue": "${OSRid}"
+      "defaultValue": "${OSRid}-${CPUArch}"
```

Comment:
https://github.com/dotnet/core-setup/pull/2342/files#diff-0b83f9dedf40d7356e5ca147a077acb4L40 is how DistroRid was forced to be portable RID when doing portable build. 

Also, DistroRid appears to be used for both portable and non-portable scenarios, so it will be good to have it in sync with what is expected.